src/pages/issues/index.tsx

```diff
@@ -47,6 +66,17 @@ const SearchField = ({ onSearchTextSubmitted, loading }: SearchFieldProps) => {
 };
 
 const Issues: FC = () => {
+    let splittedQueryParam = '';
+    const router = useRouter();
+    const dev = router?.query?.dev;
+    if (dev === 'true') {
+        const urlQueryParam = router?.query?.q as string;
```

Comment:
This is not the url query param as the name suggests, more like q part of it.

Reply:
done